templates-2.0.json

```diff
@@ -1,6 +1,81 @@
 {
 	"version":"2",
 	"templates":[
+		{
+			"type": 1,
+			"title": "FDO",
+			"description": "FDO Owner Service",
+			"categories":[
+				"edge"
+			],
+			"logo":"https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/intel.png",
+			"platform":"linux",
+			"image":"portainer/pri-fidoiot:owner-master",
+			"ports":[
+				"8042:8042/tcp",
+				"8043:8043/tcp"
+			],
+			"volumes":[
+				{
+					"container":"/home/fdo/target"
+				}
+			],
+			"note":"Owner server public URL must use the format http://IP|URL:PORT?ipaddress=IP, e.g. http://owner.portainer.io?ipaddress=183.187.112.47 or http://183.187.112.47?ipaddress=183.187.112.47",
+			"env":[
+				{
+					"name":"owner_to0_rv_blob",
+					"label": "Owner server public URL"
+				},
+				{
+					"name":"owner_api_user",
+					"label":"API user"
+				},
+				{
+					"name":"owner_api_password",
+					"label":"API password"
+				},
+				{
+					"name":"owner_to0_scheduling_interval",
+					"default":"60s",
```

Comment:
Is the `s` allowed?

Reply:
Well that's the thing, in your template you are using `60` as the value but if I read the official documentation, it specifies that the default value for `owner_to0_scheduling_interval` is `300s`: https://github.com/secure-device-onboard/pri-fidoiot/tree/2bac84514d02c89583585d3dfaa305681625c82a/component-samples/demo/owner#configuring-the-fdo-pri-owner-sample